latch_cli/snakemake/workflow.py

```diff
@@ -879,16 +886,44 @@ def build_jit_register_wrapper(cache_tasks: bool = False) -> JITRegisterWorkflow
     task_interface = Interface(
         python_interface.inputs, python_interface.outputs, docstring=None
     )
-    task = PythonAutoContainerTask[T](
-        name=f"{wrapper_wf.name}_task",
-        task_type="python-task",
-        interface=task_interface,
-        task_config=None,
+
+    primary_container = V1Container(name="primary")
+    resources = V1ResourceRequirements(
+        requests={
+            "cpu": "500m",
+            "memory": "1Gi",
+            "ephemeral-storage": "50Gi",
+        },
+        limits={
+            "cpu": "1",
+            "memory": "2Gi",
+            "ephemeral-storage": "100Gi",
+        },
```

Comment:
Seems like a lot of resources, they are kinda out of line with what you have provisioned ephemeral storage wise too -- 100m,200Mi,50Gi (req) and 200m,400Mi,100Gi (limit) are perfectly packed for your instance size (t3.medium)?